vNext/extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts

```diff
@@ -46,6 +47,11 @@ export default class PropertiesPlugin implements ITelemetryPlugin, IPropertiesPl
         }
 
         this._logger = core.logger;
+
+        if (this._extensionConfig.isCookieUseDisabled()) {
+            Util.disableCookies();
```

Comment:
You are now disabling cookies here and in Analytics extension, if some other extension use cookies in the future you will add the same code over there, it would be better to have this code in Core and disable cookies during Core initialization, in that case I can start using Cookie methods in Common library in 1p code and I don't need to worry about this config ever

Reply:
This is true. I would like to put this stuff in core, but it cannot take a dependency on common.

Reply:
there is a coreUtil where you could move the isCookieUseDisabled() functionality to. Core has access to config at init, it can set it early on.

Reply:
good call! I will move them to CoreUtils